src/store/settings.ts

```diff
@@ -61,6 +62,7 @@ function initialState() {
     selectNodeDialogVisibility: false,
     blockNumber: 0,
     blockNumberUpdates: null,
+    storageReferral: storage.get('storageReferral') || '',
```

Comment:
How do you think, could we transfer `storageReferral` to `referrals.ts` store to keep together the functionality of this feature? `settings.ts` store already has a lot of code